README.md

```diff
@@ -1 +1,31 @@
 # RestPyTAC
+Short description
+
+# Setup
+### Prerequisites 
+- Python [3.4 or higher](https://www.python.org/downloads/)
+- Virtual environment [Virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
+- Cloned repository [git](git@github.com:repo/repo.git)
+- Docker [install](https://docs.docker.com/install/)
+- Docker-compose [install](https://docs.docker.com/compose/install/)
+
+### Setting up the project
+- create new virtual env `mkvirtualenv -p <path/to/python3> project_name`, 
+you could use any other name instead of `project_name`
+- activate the env by running `workon project_name`
+- Install modules `pip install -r ~/<path to project>/requirements.pip`
+- Clone repository [rest_Spring_Docker](https://github.com/momel/rest_Spring_Docker)  
+- Up clone image: `cd ~/<path to your cloned repository>` and `docker-compose up -d`
```

Comment:
It's not clear what we actually doing with this command, consider changing wording